.github/workflows/cli-verification.yaml

```diff
@@ -28,29 +28,66 @@ jobs:
                   registry-url: 'https://npm.pkg.github.com'
                   scope: '@nangohq'
                   always-auth: true
-            - name: Publish npm packages to the github registry
+            - name: Publish packages to github registry so they can be bumped
+              env:
+                  NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+              run: |
+                  function check_and_publish {
+                    PACKAGE_PATH=$1
+                    PACKAGE_NAME=$(node -p "require('./${PACKAGE_PATH}/package.json').name")
+                    PACKAGE_VERSION=$(node -p "require('./${PACKAGE_PATH}/package.json').version")
+                    if npm view ${PACKAGE_NAME}@${PACKAGE_VERSION} ; then
+                      echo "Version ${PACKAGE_VERSION} of ${PACKAGE_NAME} already exists. Skipping publish."
+                    else
+                      echo "Publishing ${PACKAGE_NAME}@${PACKAGE_VERSION}..."
+                      pushd ./${PACKAGE_PATH}
+                      npm publish --ignore-scripts
```

Comment:
I don't see `--ignore-scripts` in the allowed flag, what does it do?

Reply:
Doesn’t run the prepublish and postbuild scripts since we don’t need them

Reply:
the prepublish does the build no, if we skip I wonder what we are publishing?

Reply:
The publishing here is more a formality so that the later the npm version command doesn’t fail.